src/math.ts

```diff
@@ -266,8 +266,9 @@ export const getGridPoint = (
   x: number,
   y: number,
   gridSize: number | null,
+  snapToGrid: boolean = false,
```

Comment:
we can reuse the `gridSize` for this, by assigning `null` if we don't want to snap to grid. Not the most intuitive parameter (as it's currently being used already), but your proposal makes the grid snapping dependent on two separate parameters so it's making it even less clear.

Reply:
I was contemplating this, but opted for making it explicit with snapToGrid. Now I changed it to null if ctrl_or_cmd is down.